packages/cli/src/commands/config/config.js

```diff
@@ -2,10 +2,31 @@
  * @flow
  */
 import {type ConfigT} from 'types';
+
+function isValidRNDependency(config) {
+  return (
+    Object.keys(config.platforms).filter(key => Boolean(config.platforms[key]))
+      .length !== 0 ||
+    Object.keys(config.hooks).length !== 0 ||
+    config.assets.length !== 0 ||
+    config.params.length !== 0
+  );
+}
+
+function filterConfig(config) {
+  const filtered = {...config};
+  Object.keys(filtered.dependencies).forEach(item => {
+    if (!isValidRNDependency(filtered.dependencies[item])) {
+      delete filtered.dependencies[item];
+    }
+  });
+  return filtered;
+}
+
 export default {
   name: 'config',
   description: 'Print CLI configuration',
   func: async (argv: string[], ctx: ConfigT) => {
-    console.log(JSON.stringify(ctx, null, 2));
+    console.log(JSON.stringify(filterConfig(ctx), null, 2));
```

Comment:
This is a good idea to do it here - we still keep the lazy behaviour in other places

Reply:
Yup, that was the intention. Should I leave a comment to make it obvious?

Reply:
I think it's fine like this :)

Reply:
Feel free to approve it then? 🙂

Reply:
Be my guest